sessions.qmd

```diff
@@ -3,108 +3,132 @@ title: "Sessions"
 comments: false
 ---
 
+# Day 1: Deconstructing epidemiological timelines
+
+**Monday June 23**
+
 ## Session 0: Introduction and course overview
 
-Monday June 24: 9.00-9.30
+Monday June 23: 09.00-09.30
 
-- Introduction to the course and the instructors (10 mins)
-- Motivating the course: From an epidemiological line list to informing decisions in real-time (20 mins)
+-   [Introduction](introduction-to-the-course-and-the-instructors.qmd): the course and the instructors (10 mins)
+-   [Motivating the course](from-line-list-to-decisions.qmd): From an epidemiological line list to informing decisions in real-time (20 mins)
 
 ## Session 1: R, Stan, and statistical concept background
 
-Monday June 24: 9.30-10.30
+Monday June 23: 09.30-10.30
 
-- Introduction to statistical concepts used in the course (15 mins)
-- Introduction to stan concepts used in the course (15 mins)
-- Practice session: introduction to estimation in stan (30 mins)
+-   [Introduction](sessions/slides/introduction-to-statistical-concepts): statistical concepts used in the course (15 mins)
```

Comment:
I thought we were going to get rid of the intro to statistical concepts and assume people come with some familiarity of Bayesian statistics?

Reply:
Yes we didn't track these decisions very well so we probably want a new issue

Reply:
#426